library/pkcs12.c

```diff
@@ -216,7 +216,10 @@ int mbedtls_pkcs12_pbe( mbedtls_asn1_buf *pbe_params, int mode,
     }
 
     if( ( ret = mbedtls_cipher_finish( &cipher_ctx, output + olen, &olen ) ) != 0 )
-        ret = MBEDTLS_ERR_PKCS12_PASSWORD_MISMATCH;
+    {
+        if( ret != MBEDTLS_ERR_DES_FEATURE_UNAVAILABLE )
```

Comment:
There is no PKCS12 error, because there is no alternative for PKCS12.
At the moment, the only PKCS12 PBE types we support are `pbeWithSHAAnd3-KeyTripleDES-CBC` and `pbeWithSHAAnd2-KeyTripleDES-CBC`, this is why the only acceptable FEATURE UNAVAILABLE I see is the DES one.

Reply:
What happens if the user tries to read some data which happens to be encrypted with AES?